### DISCUSSION TITLE
微信小程序webview中 嵌入web页面 部分安卓机型播放不了

### DISCUSSION THREAD

**Discussion by zxcv66666** [Tencent/libpag#1726] [Q&A] [answered]
微信小程序/支付宝小程序webview中 嵌入web页面 部分安卓机型播放不了视频流动画，而是只显示一张静态图，点击屏幕或者各个地方 也不能播放，想问一下这个是需要安装软解码器么，还是因为手机性能不行呢 感谢指导

**Comment by domchen** [comment, marked as answer]
这个应该是因为有些webview没开视频自动播放的权限，需要在响应交互的情况下才可以播放视频，可以参考这个文档：https://pag.art/docs/web-features.html。这个如果是自己的webview可以设置参数开启。是别人的就没办法了。只能不使用带BMP预合成的PAG。或者采用软件解码的方式，但是那个会比较卡。建议优化素材。
